Apps/W1/DynamicsSLHistoricalData/app/app.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "4f3fe3fd-bdc4-4371-8579-d53820b93575",
+  "name": "Dynamics SL Historical Data",
+  "publisher": "Microsoft",
+  "version": "25.0.0.0",
+  "brief": "This extension adds the necessary components to retain historical Dynamics SL data after the cloud migration. Only uninstall this extension if the historical data is no longer needed.",
+  "description": "This extension adds the necessary components to retain historical Dynamics SL data after the cloud migration. Only uninstall this extension if the historical data is no longer needed.",
+  "privacyStatement": "https://go.microsoft.com/fwlink/?LinkId=724009",
+  "EULA": "https://go.microsoft.com/fwlink/?linkid=2009120",
+  "help": "https://go.microsoft.com/fwlink/?linkid=2009037",
+  "contextSensitiveHelpUrl": "https://go.microsoft.com/fwlink/?linkid=2285106",
+  "url": "https://go.microsoft.com/fwlink/?LinkId=724011",
+  "logo": "MSFTBCIcon.png",
+
+  "screenshots": [],
+  "platform": "25.0.0.0",
```

Comment:
Should be version 26.

Reply:
I thought this extension was going to be made available in an upcoming version 25 update.

Reply:
These values have been updated to 26.0.0.0 with Nikola's updates that he asked me to apply to this PR. These updates are included in commit cef586f283fbbdc4554d20cd630bf5cf70ca053c